demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -904,11 +616,12 @@ static void eventCallback( MQTTContext_t * pContext,
 
                 /* Nothing to be done from application as library handles
                  * PINGRESP. */
-                LogInfo( ( "PINGRESP received.\n\n" ) );
+                LogWarn( ( "PINGRESP should not be handled by the application "
```

Comment:
Is the invocation of callback in case of PINGRESP unexpected? Is yes, it makes sense to log it as warn which indicates that it is some sort of unexpected behavior. Otherwise, it should be logged as info.

Reply:
Yes, it is unexpected in the case of invoking `ProcessLoop` instead of `ReceiveLoop`. I do agree keeping it as `LogWarn`.